pkgs/by-name/sw/swaybg/package.nix

```diff
@@ -17,13 +19,29 @@ stdenv.mkDerivation rec {
 
   strictDeps = true;
   depsBuildBuild = [ pkg-config ];
-  nativeBuildInputs = [ meson ninja pkg-config scdoc wayland-scanner ];
+  nativeBuildInputs = [ meson ninja pkg-config scdoc wayland-scanner wrapGAppsNoGuiHook ];
   buildInputs = [ wayland wayland-protocols cairo gdk-pixbuf ];
 
   mesonFlags = [
     "-Dgdk-pixbuf=enabled" "-Dman-pages=enabled"
   ];
 
+  preFixup = ''
+    makeWrapperArgs+=("''${gappsWrapperArgs[@]}")
```

Comment:
I don't see `makeWrapper` getting called anywhere?

Reply:
thanks for the feedback. i added the wrapper now.